as/itests/pom.xml

```diff
@@ -64,18 +64,76 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
 	</build>
 		
 	<profiles>
+
+		<!-- this profile can be used to simply display the values of project.build.directory and requirementsDirectory -->
+		<profile>
+			<id>verbose-requirementsDirectory</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-antrun-plugin</artifactId>
+						<version>${maven.antrun.plugin.version}</version>
+						<executions>
+							<execution>
+								<phase>verify</phase>
+								<goals>
+									<goal>run</goal>
+								</goals>
+								<configuration>
+									<quiet>true</quiet>
+									<tasks>
+										<echo>project.build.directory = ${project.build.directory}</echo>
+										<echo>requirementsDirectory = ${requirementsDirectory}</echo>
+									</tasks>
+								</configuration>
+							</execution>
+						</executions>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
 		<profile>
 			<id>download-all-runtimes</id>
-			<!-- Do not download all runtimes if only testing one runtime - eap compatibility testing -->
 			<activation>
+				<!-- JBIDE-24763 only download everything if we're running in a plugin folder, not the itests/ folder. -->
+				<file>
+					<exists>../../itests</exists>
+				</file>
```

Comment:
ok, I was thinking that perhaps we could do the opposite - only download the runtimes once in the itests/ folder and use it everywhere else. your approach is slightly slower because 3 times copying and unzipping. but I guess it's fine.

Reply:
Rob specifically requested that we do a clean unzip before each test plugin run. that way any dirty state in the runtimes is wiped and refreshed

Reply:
yeah, ok, in that case.. I guess it's possible that a failed test will leave a server dirty